README.md

```diff
@@ -1,24 +1,43 @@
-## TORSO-21 Dataset: Typical Objects in RoboCup Soccer 2021
+# TORSO-21 Dataset: Typical Objects in RoboCup Soccer 2021
 
-This repository contains the scripts and addtional information for the TORSO-21 Dataset.
-This is a dataset for the RoboCup Humanoid Soccer domain consisting of images of the Humanoid League as well as the Standard Platform League. We provide two image collections. The first one consists of images from various real-world locations, recorded by different robots. It includes annotations for the ball, goalposts, robots, lines, field edge, and three types of line intersections. The second collection is generated in the Webots simulator which is used for the official RoboCup Virtual Humanoid Soccer Competition. Additionally to the labels of the first collection, labels for the complete goal, depth images, 6D poses for all labels, as well as the camera location in the field of play, are provided.
+This repository contains the scripts and additional information for the TORSO-21 Dataset.
+This is a dataset for the RoboCup Humanoid Soccer domain consisting of images of the Humanoid League as well as the Standard Platform League. We provide two image collections. The first one consists of images from various real-world locations, recorded by different robots. It includes annotations for the ball, goalposts, robots (including team color and player number), lines, field edge, and three types of line intersections. The second collection is generated in the Webots simulator which is used for the official RoboCup Virtual Humanoid Soccer Competition. Additionally to the labels of the first collection, labels for the complete goal, depth images, 6D poses for all labels, as well as the camera location in the field of play, are provided.
 
-### Meta Data
-| # of real-world Images   | 10464 |
+## Meta Data
+### Real World
+| # of Images              | 10464 |
 |--------------------------|-------|
-| # of Balls               | 10959 |
-| # of Robots              | 14383 |
-| # of Goalposts           | 12780 |
-| # of L-Intersections     | 15458 |
-| # of T-Intersection      | 13479 |
-| # of X-Intersections     | 13447 |
+| # of Balls               |  6081 |
+| # of Robots              |  7641 |
+| # of Goalposts           |  7888 |
+| # of L-Intersections     | 10375 |
+| # of T-Intersection      |  8659 |
+| # of X-Intersections     |  7268 |
 | # of Field Segmentations | 10464 |
```

Comment:
I guess, before those numbers represented counts of that object type and now its count of images with that object? We should make that clear somehow

Reply:
Based on the previous # of robots, I assume it was the number of annotations of type xy irrespective of the value of the "in_image" attribute. Now, it is the number of annotations of type xy having "in_image" set to true.

Reply:
Maybe we should recalculate the segmentation numbers too? I think not all images have lines visible.

Reply:
I think a blank segmentation still counts as a segmentation label so I would keep them.